packages/terra-table/package.json

```diff
@@ -28,8 +28,10 @@
     "classnames": "^2.2.5",
     "keycode-js": "^2.0.1",
     "prop-types": "^15.5.8",
+    "terra-content-container": "^3.14.0",
     "terra-doc-template": "^2.22.0",
```

Comment:
If you do convert over to mdx, you should be able to drop the terra-doc-template dependency.